repos/fdbt-site/tests/pages/api/csvUpload.test.ts

```diff
@@ -21,14 +21,22 @@ import {
 } from '../../../src/constants/attributes';
 import { ReturnTicket, SingleTicket, WithIds } from 'fdbt-types/matchingJsonTypes';
 import * as userData from '../../../src/utils/apiUtils/userData';
+import * as auroradb from '../../../src/data/auroradb';
 
+jest.mock('../../../src/data/auroradb');
 jest.spyOn(s3, 'putDataInS3');
 jest.spyOn(userData, 'putUserDataInProductsBucketWithFilePath');
 
 describe('csvUpload', () => {
     const loggerSpy = jest.spyOn(logger, 'warn');
     const getFormDataSpy = jest.spyOn(fileUpload, 'getFormData');
     const updateSessionAttributeSpy = jest.spyOn(sessions, 'updateSessionAttribute');
+    const updateProductFareTriangleModifiedByNocCodeAndIdSpy = jest.spyOn(
+        auroradb,
```

Comment:
Do we have any new tests for the new API functionality?

Reply:
no as we are just updating the current date in products table for fareTriangleModified